proof/refine/ARM_HYP/KHeap_R.thy

```diff
@@ -713,9 +702,9 @@ lemma cte_wp_at_ctes_of:
    apply (simp add: field_simps)
   apply (clarsimp split: if_split_asm del: disjCI)
    apply (simp add: ps_clear_def3 field_simps)
-  apply (rule disjI2, rule exI[where x="(p - (p && ~~ mask 9))"])
+  apply (rule disjI2, rule exI[where x="p - (p && ~~ mask 9)"])
```

Comment:
I'm curious why this didn't need parens for `~~ (mask 9)`, when that seems to have been necessary in lots of other places?

Reply:
The other places with the many changes were synced from the AFP where that change was applied globally once, but I didn't undo it globally when I reverted the syntax change (the real change Florian wanted was for `NOT`, but doing the same for `~~` was a bit too much, the syntax precedence we have is analogous to other symbolic not-operators).

Reply:
See @lsf37's reply to my comment on `lib/Word_Lib/Aligned.thy`, which answers this.